web/src/pages/Route/List.tsx

```diff
@@ -90,6 +101,47 @@ const Page: React.FC = () => {
       dataIndex: 'desc',
       hideInSearch: true,
     },
+    {
+      title: formatMessage({ id: 'component.global.label' }),
+      dataIndex: 'labels',
+      render: (_, record) => {
+        return Object.keys(record.labels || {}).map((item) => (
+          <Tag key={Math.random().toString(36).slice(2)}>
+            {item}:{record.labels[item]}
```

Comment:
just to make sure that if the `labels` we got from api is a `[key:string]:[value:string]` object